Core/FeatureFlag.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  FeatureFlag.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import BrowserServicesKit
+
+public enum FeatureFlag: String {
+    case debugMenu
+    case sync
+    case credentialsAutofill
+    case credentialsSaving
+    case inlineIconCredentials
+    case accessCredentialManagement
```

Comment:
What do you think about naming these such that it's clear that they're all autofill related? It seems obvious now, but once we have a bunch more of these I could see it becoming messy.